src/Host.h

```diff
@@ -161,6 +161,7 @@ class Host  : public QObject
     bool               mAllowToSendCommand;
     bool               mAutoClearCommandLineAfterSend;
     bool               mBlockScriptCompile;
+    bool               mEchoLuaErrors; 
```

Comment:
I know I had it being initialized as false at some point when I was making it, just not sure at what point I accidentally undid that change. Hrm...